krun.py

```diff
@@ -203,13 +200,41 @@ def get_overall_time_estimate_formatter(self):
     def add_eta_info(self, key, exec_time):
         self.eta_estimates[key].append(exec_time)
 
+    def build_schedule(self, config, current):
```

Comment:
Sorry, I used the wrong language. That is exactly what I thought it was. The results file contains metadata too. So why not call it `current_result_json`?